clusterloader2/testing/load/modules/scheduler-throughput.yaml

```diff
@@ -0,0 +1,113 @@
+## Input params
+# Valid actions: "create", "delete"
+{{$action := .action}}
+{{$namespaces := .namespaces}}
+{{$replicasPerNamespace := .replicasPerNamespace}}
+{{$schedulerThroughputNamespaces := .schedulerThroughputNamespaces}}
+{{$schedulerThroughputPodsPerDeployment := .schedulerThroughputPodsPerDeployment}}
+
+## Derivative variables
+{{$is_creating := (eq .action "create")}}
+{{$is_deleting := (eq .action "delete")}}
+
+## CL2 params
+{{$ENABLE_HUGE_SERVICES := DefaultParam .CL2_ENABLE_HUGE_SERVICES false}}
+{{$SCHEDULER_THROUGHPUT_THRESHOLD := DefaultParam .CL2_SCHEDULER_THROUGHPUT_THRESHOLD 100}}
+
+steps:
+{{if $is_creating}}
+- name: Creating scheduler throughput measurements
+  measurements:
+  - Identifier: HighThroughputPodStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: start
+      labelSelector: group = scheduler-throughput
+      threshold: 1h # TODO(https://github.com/kubernetes/perf-tests/issues/1024): Ideally, this should be 5s
+  - Identifier: WaitForSchedulerThroughputDeployments
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: start
+      apiVersion: apps/v1
+      kind: Deployment
+      labelSelector: group = scheduler-throughput
+      # The operation timeout shouldn't be less than 20m to make sure that ~10m node
+      # failure won't fail the test. See https://github.com/kubernetes/kubernetes/issues/73461#issuecomment-467338711
+      operationTimeout: 20m
+  - Identifier: SchedulingThroughput
+    Method: SchedulingThroughput
+    Params:
+      action: start
+      labelSelector: group = scheduler-throughput
+      measurmentInterval: 1s
+  {{if $ENABLE_HUGE_SERVICES}}
```

Comment:
I'm not a big fan of using "if is_creating" pattern -- technically we can simply take old configs put them here with if condition, but it doesn't reduce code complexity.

I think we can avoid doing that for "Creating huge services" step which (correct me if I'm wrong) can be implemented in a ccommon way for both creation and deletion paths (the only diff is replicasPerNamespace and acction name).

You can use the same pattern as for "{{$action}} scheduler throughput pods" step

Reply:
Ohh, I see why it's needed -- the order in creation and deletion is different here.

Reply:
Yeah, it was a bit upsetting for me too. :/